metadata.json

```diff
@@ -50,7 +50,8 @@
     {
       "operatingsystem": "OpenBSD",
       "operatingsystemrelease": [
-        "6.2"
+        "6.6",
```

Comment:
mhm, is this a backwards incompatible change if we drop 6.2?

Reply:
i reckon that the latest supported versions are the last two, so it's more like a cleanup

Reply:
Indeed. I think it also didn't really break compatibility with 6.2. That's just an EOL OS.